android/settings.gradle.kts

```diff
@@ -0,0 +1,23 @@
+include(":core")
+project(":core").projectDir = file("../core")
+
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        jcenter()
+        google()
+    }
+    resolutionStrategy {
+        eachPlugin {
+            if (requested.id.id.startsWith("com.android")) {
+                useModule("com.android.tools.build:gradle:3.5.3")
```

Comment:
Can we access the Dependencies.kt file in here to reference the android gradle plugin version?

If no, then I would prefer to remove the android gradle plugin version in Dependencies.kt. Because android version only needs to be specified once which could be here. 
The plugins like com.android.application will then automatically use the version of the resolutionStrategy as far as I know.

Reply:
Ok i messed up there this was supposed to be `${requested.version}` as we can access Dependencies.kt in build.gradke.kts